airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2018,6 +2034,13 @@ components:
       enum:
         - full_refresh
         - incremental
+    DestinationSyncMode:
+      type: string
+      enum:
+        - append
+        - overwrite
+        - upsert_dedup
+        - append_dedup
```

Comment:
which ones of these are you planning to implement as part of this project? i'd suggest we only add the ones we actually plan to implement now so that people don't read this and think we support something we don't. if you wanted to leave the others as comments as a teaser that would be okay.

Reply:
okay. i'm not sure exactly what the last 2 mean. upsert_dedup is SCD1 and append_dedup is SCD2? i'll just point out that achieving just one of these will make this project successful, so if trying to go for both makes it riskier it may not be worthwhile to include it yet. up to you though!

Reply:
Got it!